### PR TITLE
Allow passing of axios_config

### PR DESCRIPTION
**Pull request by ueda000**
## Description

Fixes closed PR https://github.com/LuanRT/google-this/pull/29

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: lib/core/main.js [hunk 1]
```diff
@@ -31,7 +31,8 @@ const FormData = require('form-data');
  * @param {number} [options.page] - Page number.
  * @param {boolean} [options.parse_ads] - Weather or not to parse ads.
  * @param {boolean} [options.use_mobile_ua] - Weather or not to use a mobile user agent.
- * @param {object} [options.additional_params] - parameters that will be passed to Google
+ * @param {object} [options.additional_params] - Additional parameters that will be passed to Google.
```
Comment (ueda000) [inline lib/core/main.js:34]: refactoring: aligned with [other function](https://github.com/LuanRT/google-this/pull/38/files#diff-c8065054c65fe77edfce585a229adcd8d39fbcd0258aa3f4791b4063aa1fda6eR146)

File: lib/core/main.js [hunk 2]
```diff
@@ -41,10 +42,11 @@ async function search(query, options = {}) {
   const page = options.page ? options.page * 10 : 0;
   const use_mobile_ua = Reflect.has(options, 'use_mobile_ua') ? options.use_mobile_ua : true;
   const parse_ads = options.parse_ads || false;
-  const additional_params = options.additional_params || null;
+  const additional_params = options.additional_params || {};
```
Comment (ueda000) [inline lib/core/main.js:45]: refactoring: aligned with [other function](https://github.com/LuanRT/google-this/pull/38/files#diff-c8065054c65fe77edfce585a229adcd8d39fbcd0258aa3f4791b4063aa1fda6eR146)

**Comment by ueda000** [comment]
@LuanRT 
This is fix of https://github.com/LuanRT/google-this/pull/29.
I have confirmed the operation in my environment.

**Review by LuanRT — APPROVED** [review]

**Comment by LuanRT** [comment]
Awesome, many thanks!

**Pull request merged**